src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.PowerShell.Commands
     /// The ConvertTo-Json command.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
ConvertTo-Json.Should handle null
```powershell
Value cannot be null. (Parameter 'type')
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\ConvertTo-Json.Tests.ps1: line 88
88:         $null | ConvertTo-Json -Compress | Should -Be 'null'

```